lib/dai-plugin-mcd/src/ManagedCdp.js

```diff
@@ -32,6 +30,78 @@ export default class ManagedCdp {
     return art.times(rate).shiftedBy(-27);
   }
 
+  async getCollateralizationRatio() {
+    const [collateral, debt, price] = await Promise.all([
+      this.getCollateralValue(),
+      this.getDebtValue(),
+      this._cdpType.getPrice()
+    ]);
+
+    return (
+      collateral
+        .times(price)
+        .div(debt)
+        .toNumber() * 100
+    ).toFixed(2);
+  }
```

Comment:
I feel like removing the toFixed(2) here would make sense, so that the user can decide how many decimals to keep.  Also, the return value here is in percentage format (eg. 150), while getLiquidationRatio's is not (e.g 1.5).  I think the formats should probably be consistent between the two.  I don’t have a strong opinion on which should be changed though

Reply:
agreed

Reply:
actually let's drop the toNumber() here as well, so that you return a CurrencyRatio object of type USD per COL. this will help validate your math in other methods that use this. you should read [this spec](https://github.com/makerdao/currency/blob/master/test/index.spec.js) to understand how Currency and CurrencyRatio objects work.

Reply:
+1